lumps/dehacked/dehacked.txt

```diff
@@ -144,25 +144,25 @@ STARTUP5========================================================================
 #endif
 GOTARMOR = Picked up some basic infantry armor.
 GOTMEGA = Picked up some heavy assault armor.
-GOTHTHBONUS = Got a 1% health bonus.
-GOTARMBONUS = Got a 1% armor bonus.
+GOTHTHBONUS = Picked up a life charge.
+GOTARMBONUS = Picked up an armor charge.
 GOTSTIM = Picked up a small health pack.
-GOTMEDINEED = Found a large health pack, just in time!
+GOTMEDINEED = Picked up a large health pack, just in time!
 GOTMEDIKIT = Picked up a large health pack.
-GOTSUPER = You got the overdrive sphere!
+GOTSUPER = Hundredfold life!
```

Comment:
Doesn't "hundredfold" mean "multiplied by 100"?

Reply:
It used to be "Hundredfold life charge" but that got too bleh so I cut out the last word.